migrations/migrations/20210628172053-create-table-users.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      first_name: {
+        type: Sequelize.STRING,
+        defaultValue: false,
```

Comment:
There's no need to add this

Reply:
Does the Sequelize data type default to String ? or do you mean `defaultValue: false` ?

Reply:
Solved, thanks!